### ISSUE TITLE
Add request rate limit to proxy

### ISSUE THREAD

**Issue by ai** [hplush/slowreader#168]
**Blocked by #100**

To prevent abusing our public CORS proxy

- [ ] Allow only 500 request in a minute to the same domain from the same IP-address.
- [ ] Allow only 5000 request to any domain from the same IP-address.

On passing those limits:
1. Print to console (we will need it to debug)
2. Do not cancel response, just postpone (so client will not see an error, just slower connection).

These changes need to be done in `proxy/`. It will be cool to have tests, but if they are not stable or hard to make, we can avoid it.

**Comment by toplenboren** [comment]
> Allow only 5000 request to any domain from the same IP-address.

What if I really like to read someone's blog :-) shall we reset the counter monthly?

**Comment by janefawkes** [comment]
hi, can i take this one?
